README.md

```diff
@@ -51,12 +52,20 @@ Linter was applied so as to have style bug-free codes.
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
- <summary>Form section</summary>
-  
- Improving the portfolio by working on :
- - **[Contact-me form]**
+ <summary><h3>Desktop version</h3></summary>
+  <p>Implementation of the desktop versio of the portfolio to give a better User's experience.</p>
  
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```

Comment:
[Optional] Please update the readme file with the necessary information relevant to this project. Avoid using placeholder text or template content. Instead, provide specific and accurate details about the project, Please remove this placeholder text.